### ISSUE TITLE
SeAT won't start due to inability to connect to TS3

### ISSUE THREAD

**Issue by sgeine** [warlof/seat-connector#15]
Thank you for breaking out the server ip/host to the config. adding prefixing for usernames was also great. Unfortunately, it appears 11001 is the default server query port rather than 6379 (which is actually redis). as a result SeAT won't start at all.
<img width="325" alt="Screen Shot 2019-09-06 at 12 11 13 AM" src="https://user-images.githubusercontent.com/48392830/64467943-20e3fd00-d0d3-11e9-9c17-c461b3e1eafc.png">

and I'm unable to find where this config has been included via recursive 'find' or 'grep'
<img width="488" alt="Screen Shot 2019-09-06 at 6 32 24 PM" src="https://user-images.githubusercontent.com/48392830/64468064-b8961b00-d0d4-11e9-8abc-3a839f6bb05c.png">
where is this config stored so I can correct it?

**Comment by warlof** [comment]
Hi,

6379 isn't related to teamspeak, it's the redis port.
Ensure redis service or redis container is started.

It's a core settings which can be change inside `.env` file https://github.com/eveseat/seat/blob/master/.env.example#L22

**Comment by sgeine** [comment]
Yes, 6379 isn't related to teamspeak, it was the default port that appeared with the teamspeak connector and I didn't change it. anyway, now that the port is incorrect for server query in teamspeak connector seat won't start. I cannot find which file to correct the teamspeak connector port. I can't do it in the web interface since it's a fatal seat error that prevents it from starting. redis is fine. the error I posted above is a teamspeak connector error. the teamspeak server connector port does not appear to be in the .env file.

**Comment by warlof** [comment]
Can you give me the full error stack please?

**Comment by sgeine** [comment]
@warlof thanks so much for your reply. I was able to find the problem it just happened to coincide with this update so that stuff is difficult to separate. not using enabling IPv6 on the server broke Redis recently (it's never been on, on my server) which broke SeAT which is reliant on Redis. once I cleared the ipv6 requirement for Redis in redis.conf I was able to solve the problem. further registration for teamspeak connector now works correctly. I'm not sure if it's supposed to open teamspeak upon successful registration (it doesn't) but registration works at least.

Thanks again or all your help!
